federation/pkg/federation-controller/sync/replicasetcontroller_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package replicaset
+package sync
```

Comment:
I'd hope the numbers adding up could be validated separately from propagation, but sure.